examples/controller-runtime/main.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
This is only an example code showing how to use the library. I used it to test if it works. It's a separate module so doesn't pollute the dependency tree.